tests/benchmarks/sys_kernel/src/mem_slab.c

```diff
@@ -0,0 +1,115 @@
+/* memslab.c */
+
+/*
+ * Copyright (c) 2020, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "syskernel.h"
+
+#define MEM_SLAB_BLOCK_SIZE  (32)
+#define MEM_SLAB_BLOCK_CNT   (10240)
```

Comment:
this is probably too aggressive as most boards won't have enough ram. `BLOCK_SIZE` could be 8. You can play with block count to get fairly consistent numbers. Maybe 500 would be enough.

Reply:
i tried 8/256 and modified a bit test to give avg time per single call and it seems that results are fairly comparable with 512 or 1024 iterations.

Reply:
I modified a test a little. Now it runs `number_of_loops`, which is more obvious. Also reduced the size of single chunk in slab to `8`.